tests/test_display.py

```diff
@@ -841,15 +842,15 @@ def test_same_axes(x_axis, y_axis, xlim, ylim, out):
 
 def test_auto_aspect():
```

Comment:
> ie in this case, it would be -k test_auto_aspect. This will generate a new baseline image stored under the tmp/ folder, which can be copied over to tests/baseline_images/test_display/ (assuming it looks correct) and added to the PR.

This test is not decorated with `mpl_image_compare`, so I thought maybe image comparison is not needed. The input data is also very mock-ish (`np.zeros((12, 12))`) and does not produce a useful output.

Reply:
oh fair enough!  Upon a closer inspection, this test works entirely on the axis properties and not on the rendered image.  Please disregard my earlier comment.